### PR TITLE
PROD changes

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: dags/infra/s3_buckets.py [hunk 1]
```diff
@@ -29,6 +22,15 @@
...
 )  # qa
 
+# #################### SENTINEL - 1  ####################
+SENTINEL_1_SYNC_BUCKET_NAME = Variable.get("sentinel_1_sync_bucket_name", "")  # qa
+
+SENTINEL_1_INVENTORY_BUCKET_NAME = Variable.get(
+    "sentinel_1_inventory_bucket_name", ""
+)  # qa
+
+SENTINEL_1_SYNC_BUCKET_NAME = Variable.get("sentinel_1_sync_bucket_name", "")  # qa
```
Comment (NikitaGandhi) [inline dags/infra/s3_buckets.py:32]: duplicate

File: dags/infra/variables.py [hunk 2]
```diff
@@ -10,6 +10,11 @@
     "deafrica_sentinel_2_cogs_folder", "sentinel-s2-l2a-cogs"
 )
 
+S3_VPC_ENDPOINT = Variable.get(
```
Comment (NikitaGandhi) [inline dags/infra/variables.py:13]: Is this used?

File: dags/infra/s3_buckets.py (hunk 1 above)
Comment (carioca-au) [reply dags/infra/s3_buckets.py]: Fixed

File: dags/infra/variables.py (hunk 2 above)
Comment (carioca-au) [reply dags/infra/variables.py:13]: Removed

**Review by NikitaGandhi — APPROVED** [review]

**Pull request merged**